apps/astro-blog/src/app.css

```
@@ -1,66 +1,851 @@
   overflow-y: scroll;
+  background: var(--bg);
+}
+
+body {
+  min-height: 100vh;
+  background:
+    radial-gradient(circle at top left, rgba(198, 255, 59, 0.2), transparent 28%),
+    radial-gradient(circle at 100% 15%, rgba(255, 142, 60, 0.12), transparent 18%),
+    linear-gradient(180deg, #f5efe6 0%, #ece4d9 48%, #e3dbd1 100%);
+  color: var(--text);
+  font-family: 'Noto Sans JP', 'Hiragino Sans', 'Yu Gothic', 'Meiryo', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Meiryo" (font-family-name-quotes)

(font-family-name-quotes)